e2e/tests/enrollment.spec.ts

```diff
@@ -40,6 +41,47 @@ test.describe('Create user with enrollment enabled', () => {
     dockerDown();
   });
 
+  test('Try to complete enrollment with disabled user', async ({ page, browser }) => {
+    await disableUser(browser, user);
+    expect(token).toBeDefined();
+    await waitForBase(page);
+    await page.goto(testsConfig.ENROLLMENT_URL);
+    await waitForPromise(2000);
+    // Test if we can send the token
+    await selectEnrollment(page);
+    await setToken(token, page);
+    await expect(page.getByText('Field is invalid')).toBeVisible();
+    // Check if we are still on the token page
+    expect(page.url()).toBe(`${testsConfig.ENROLLMENT_URL}/token`);
+    await enableUser(browser, user);
+    await page.reload();
+    await setToken(token, page);
+    // Welcome page
+    await page.getByTestId('enrollment-next').click();
+    // Data validation
+    await validateData(user, page);
+    await page.getByTestId('enrollment-next').click();
+    await disableUser(browser, user);
+    // Set password
+    await setPassword(page);
+    // VPN
+    await page.getByTestId('enrollment-next').click();
+    const deviceCreationMessage = page.waitForEvent('console');
```

Comment:
Can we find a more reliable way of capturing this event? Perhaps check if api response status == 400?